services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -725,7 +725,7 @@ protected RoomInfo roomInfo(String id, String title, String[] members, String ca
       }
     }
     room.setProfileLink(IdentityInfo.EMPTY);
-    room.setAvatarLink(LinkProvider.SPACE_DEFAULT_AVATAR_URL);
+    room.setAvatarLink(LinkProvider.ROOM_DEFAULT_AVATAR_URL);
```

Comment:
the constant shouldn't be place in social but here where it's used, even better, define a default room icon inside web-conferencing which has to be independent from chat